dhis-2/dhis-services/dhis-service-core/src/test/java/org/hisp/dhis/program/notification/ProgramNotificationServiceTest.java

```diff
@@ -658,13 +661,13 @@ void testScheduledNotifications() {
               return new BatchResponseStatus(Collections.emptyList());
```

Comment:
Can we delete this mock test which is what we generally try to do and rely on dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/program/notification/ProgramNotificationServiceIntegrationTest.java instead

Reply:
yes. I will do that after creating integration tests for `ProgramNotificationService`